problems/arraysWithLoops.js

```diff
@@ -66,15 +113,27 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
-
+function findSecondSmallest(nums) {
+  let secondSmallest = nums[0];
+  for(let i = 0; i <= nums.length; i++){
+    if (nums[i] < secondSmallest) secondSmallest = nums[i];
+  }
+  return secondSmallest + 1
+}
```

Comment:
This shouldn't work. 
The fact that it's working is a coincidence. Same with the one below. 
secondSmallest should not always be 1 plus the smallest. What if the array was [3, 5 9]

Reply:
Please revisit this problem and the one below